comfyui_custom_nodes/webui_io.py

```diff
@@ -17,7 +18,8 @@ def INPUT_TYPES(cls):
     CATEGORY = "webui"
 
     def get_images(self, void):
-        return torch.stack(global_state.node_inputs).permute(0, 2, 3, 1),
+        return ((torch.stack(global_state.node_inputs) if isinstance(global_state.node_inputs, list) else global_state.node_inputs)
```

Comment:
`global_state.node_inputs` will make multiple ipc requests. consider fetching it once and reuse the local value.